public/mobile-app/src/lib/api-holidays.ts

```diff
@@ -0,0 +1,70 @@
+import { PUBLIC_API_URL } from '$env/static/public'
+
+export type Holiday = {
+  description: string
+  zones: string
+  start_date: Date
+  end_date: Date
+  emoji: string
+}
+
+export const dayName = (date) =>
+  date.toLocaleDateString('fr-FR', { weekday: 'short' }).replace('.', '')
+
+const capitalizeFirstLetter = (val) =>
+  String(val).charAt(0).toUpperCase() + String(val).slice(1)
+
+export const monthName = (date) => {
+  return capitalizeFirstLetter(date.toLocaleDateString('fr-FR', { month: 'long' }))
+}
+
+export const holidayPeriod = (startDate, endDate) => {
+  const locale = 'fr-FR'
+  let startFormat = { year: 'numeric', month: 'long', day: 'numeric' }
+  let endFormat = startFormat
+  if (startDate.getYear() == endDate.getYear()) {
+    startFormat = { month: 'long', day: 'numeric' }
+    if (startDate.getMonth() == endDate.getMonth()) {
+      startFormat = { day: 'numeric' }
+    }
+  }
+  const start = startDate.toLocaleDateString(locale, startFormat)
+  const end = endDate.toLocaleDateString(locale, endFormat)
+  return `Du ${start} au ${end}`
+}
+
+export const retrieveHolidays = async (date) => {
+  var today = date || new Date()
+  today.setHours(0, 0, 0, 0)
+  const start_date = today.toLocaleDateString('sv-SE') // this gives the locale date in ISO format ...
```

Comment:
Wow that's crazy! Does it mean this works better than [toISOString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)?

Reply:
toIsoString gives us a datetime ISO format :/